docs/setting-up/server/aws.md

```diff
@@ -67,13 +67,13 @@ Your instance settings are summarized in a special area. Click the Launch with 1
 
 Your clicking the *Launch with 1 click* button, deploys your instance. To continue setting up your instance, run the EC2 console. It is available as a link at the top of the page that opens after you click the *Launch with 1 click* button.
 
-Your instance appears in the EC2 console in a table that lists all instances available to you. When a new instance is only created, it has no name. Make sure that you give it a name to distinguish from other instances managed via the EC2 console.
+Your instance appears in the EC2 console in a table that lists all instances available to you. When a new instance is only created, it has no name. Make sure that you give it a name to distinguish it from other instances managed via the EC2 console.
 
 ![!image](../../_images/aws-marketplace.ec2-console.pmm.1.png)
 
 ## Running the instance
 
-After you add your new instance it will take some time to initialize it. When the AWS console reports that the instance is now in a running state, you many continue with configuration of PMM Server.
+After you add your new instance ,it will take some time to initialize it. When the AWS console reports that the instance is now in a running state, you may continue with configuration of PMM Server.
```

Comment:
```suggestion
After you add your new instance, it will take some time to initialize it. When the AWS console reports that the instance is now in a running state, you may continue with the configuration of the PMM Server.
```